biz.aQute.repository/test/aQute/p2/export/P2PublisherTest.java

```diff
@@ -0,0 +1,66 @@
+package aQute.p2.export;
+
+import static org.junit.Assert.assertTrue;
```

Comment:
You are mixing JUnit 4 assertions with JUnit 5 annotated test methods. You should use all the same JUnit version (e.g. 5).